terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -91,12 +91,14 @@ receivers:
   webhook_configs:
   - send_resolved: false
     url: "${verify_joint_cronitor}"
-- name: "autom8-slack"
-  slack_configs:
+- name: "verify-2ndline-slack"
+  slack_configs: &verify-2ndline-slack-configs
   - send_resolved: true
     channel: '#verify-2ndline'
     icon_emoji: ':verify-shield:'
     username: alertmanager
 - name: "verify-p1"
   pagerduty_configs:
     - service_key: "${verify_p1_pagerduty_key}"
+  slack_configs: *verify-2ndline-slack-configs
```

Comment:
Hmm, do YAML anchors not work in `.tpl` files even though they're actually YAML, or does Terraform not work like Concourse, ie doesn't expand them in the plan?

`terraform plan` output:

```
name: \"verify-p1\"\n  pagerduty_configs:\n    - service_key: \"[REDACTED]\"\n  slack_configs: *verify-2ndline-slack-configs\n\n"
```

Reply:
YAML anchors aren't a templating syntax; they're interpreted by alertmanager when it reads the config file.

Reply:
>  or does Terraform not work like Concourse, ie doesn't expand them in the plan?

so yes i would argue that concourse has the unusual behaviour here (even if it is helpful)

Reply:
Oh, interesting. 👍